src/utils/swapErrorToUserReadableMessage.tsx

```diff
@@ -46,7 +46,7 @@ export function swapErrorToUserReadableMessage(error: any): string {
       if (reason?.indexOf('undefined is not an object') !== -1) {
         return t`An error occurred when trying to execute this swap. You may need to increase your slippage tolerance. If that does not work, there may be an incompatibility with the token you are trading. Note: fee on transfer and rebase tokens are incompatible with Uniswap V3.`
       }
-      return t`Unknown error${reason ? `: "${reason}"` : ''}. Try increasing your slippage tolerance.
-Note: fee on transfer and rebase tokens are incompatible with Uniswap V3.`
+      return t`${reason ? reason : 'Unknown error'}. Try increasing your slippage tolerance.
+Note: certain tokens are incompatible with Uniswap V3.`
```

Comment:
Should this say "fee on transfer and rebase" or just "certain" to be more broad?